lib/AST/ASTVerifier.cpp

```diff
@@ -2426,7 +2426,7 @@ class Verifier : public ASTWalker {
           abort();
         }
       }
-      if (auto addressor = ASD->getAccessor(AccessorKind::MutableAddress)) {
+      if (auto addressor = ASD->getParsedAccessor(AccessorKind::MutableAddress)) {
```

Comment:
These changes don't seem right.  The AST verifier is unlike most clients in that it should be doing verification of accessors if they exist for any reason, and I don't think we want to be embedding assumptions in it about whether specific accessors only exist if they're parsed.

Presumably we visit all the accessors that exist as part of the walk?  (That seems right for the AST verifier even if it's not normally a good operation to provide.)  Maybe these checks can be moved to when we visit the accessor.

Reply:
I can change this back. I was planning on keeping something like getAccessor() around specifically for the verifier (and maybe even friend the verifier so people don’t use it on accident).